py/server/deephaven/table_listener.py

```diff
@@ -528,13 +532,18 @@ def start(self, do_replay: bool = False, replay_lock: Literal["shared", "exclusi
             raise RuntimeError("Attempting to start an already started merged listener..")
 
         try:
-            if do_replay:
-                _do_locked(self.tables[0].update_graph, self.merged_listener_adapter.replay, lock_type=replay_lock)
+            with update_graph.auto_locking_ctx(self.tables[0]):
```

Comment:
Just a note, using the first table is only OK because you are requiring them all to be refreshing. Otherwise, it would be bad.

Reply:
A comment should be added on this detail